### DISCUSSION TITLE
how to close jobRunr version check

### DISCUSSION THREAD

**Discussion by Fionaaialong** [jobrunr/jobrunr#693] [Q&A] [answered]
Unable to check for new jobRunr version: api.github.com  ,I want to close the check verson job ,but can not find where to turn it off..Looking forward to your reply ,many thanks

**Comment by rdehuyss** [comment, marked as answer]
I'm afraid that's not possible in the free version.

In the past (and even now), too many github issues are created with people not running the latest JobRunr version (so their issue is often already resolved in a later version).

This at least makes them aware that there is a new version.
